recipes/rmats2sashimiplot/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: rmats2sashimiplot
+  version: "2.0.0"
+
+source:
+  git_rev: c3690679de217238b47251c574b3ec2b89af5745
```

Comment:
Consider using tagged releases that are accessible as tarballs from the “releases” section of the github repo.
Also don't forget to add a hash, like `md5` or `sha256`.